From: Simon McVittie <smcv@collabora.com>
Date: Mon, 30 Apr 2018 14:50:37 +0100
Subject: make-test-runtime: Look in /{usr/,}sbin for ldconfig

Under normal circumstances ldconfig isn't required to be in ordinary
users' PATHs, but running this script is not a normal circumstance.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: https://github.com/flatpak/flatpak/pull/1630
---
 tests/make-test-runtime.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/make-test-runtime.sh b/tests/make-test-runtime.sh
index 43f0997..a62360d 100755
--- a/tests/make-test-runtime.sh
+++ b/tests/make-test-runtime.sh
@@ -20,6 +20,10 @@ EOF
 
 cat ${DIR}/metadata
 
+# On Debian derivatives, /usr/sbin and /sbin aren't in ordinary users'
+# PATHs, but ldconfig is kept in /sbin
+PATH="$PATH:/usr/sbin:/sbin"
+
 # Add bash and dependencies
 mkdir -p ${DIR}/usr/bin
 mkdir -p ${DIR}/usr/lib
